envelope.json

```diff
@@ -2,10 +2,9 @@
   "$schema": "https://json-schema.org/draft/2020-12/schema",
   "type": "object",
   "title": "Envelope",
-  "description": "An envelope containing a message, the message type, and the timestamp when it was sent.",
+  "description": "An envelope containing a message and the timestamp when it was sent.",
   "required": [
     "sent_at",
```

Comment:
Woud like this to change to `produced_at` 😅 always when i see this my brains asking repateldy from where?

Reply:
Prepositions! They are confusing. In English, one sends **from** a place and sends **at** a time. It could also be called, as you suggest, `produced_at` or even `captured_at`, but then we lose the "letter/envelope/message" metaphor. One does not usually talk about letters being "captured" or "produced".